modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -118,7 +118,12 @@ v = f_y*y'' + c_y
 tangential distortion coefficients. \f$s_1\f$, \f$s_2\f$, \f$s_3\f$, and \f$s_4\f$, are the thin prism distortion
 coefficients. Higher-order coefficients are not considered in OpenCV.
 
-The next figures show two common types of radial distortion: barrel distortion (typically \f$ k_1 < 0 \f$) and pincushion distortion (typically \f$ k_1 > 0 \f$).
+The next figures show two common types of radial distortion: barrel distortion 
+(1 + k_1 r^2 + k_2 r^4 + k_3 r^6 monotonically decreasing) and pincushion distortion (1 + k_1 r^2 + k_2 r^4 + k_3 r^6 monotonically increasing). 
```

Comment:
@catree Could you please take a look on this update? Thank you!

Reply:
Yes, I think it is correct.

Most likely since `k1` is an order of magnitude greater than the other terms, it is "sufficient" (or at least this is what can be read elsewhere) to look at the sign of `k1` to determine quickly if the distortion is barrel or pincushion?

---

@midjji

Looking at the equations above, is it not if `(1 + k1.r^2 + k2.r^4 + k3.r^6)/(1 + k4.r^2 + k5.r^4 + k6.r^6)` (ignoring the tangential distortion coefficients) is monotonically decreasing it produces a barrel distortion? Or it is more complicated when taking into account the denominator?